raveandroid/src/main/res/values/strings.xml

```diff
@@ -30,6 +30,8 @@
     <string name="expiration">Expiration</string>
     <string name="email">Email</string>
     <string name="enter_your_billing_address_details">Enter your billing address details</string>
+    <string name="use_a_new_card"><u>Use another card</u></string>
```

Comment:
@jeremiahVaris Remove the underline tag. Use spans instead to decorate the text programmatically